src/west/util.py

```diff
@@ -91,6 +91,30 @@ class WestNotFound(RuntimeError):
     '''Neither the current directory nor any parent has a West installation.'''
 
 
+def get_pwd_or_cwd():
+    '''To align with shells that support symbolic links, prefer the PWD
+    environment variable, otherwise return os.getcwd()
+    '''
+    pwd = os.environ.get('PWD', None)
+    cwd = os.getcwd()
+
+    if pwd is None:
+        return cwd
+
+    # west_topdir is crucial, so let's be paranoid.
+    if not os.path.isdir(pwd):
+        raise os.error("PWD %s is not a directory" % pwd)
+
+    if not os.path.realpath(pwd) == os.path.realpath(cwd):
+        # This can happen if we're not invoked by the shell directly but
+        # by a process which changed cwd without changing PWD. To
```

Comment:
And here?

Reply:
As `/usr/bin/pwd -L` merely returns $PWD I don't see what difference it would make in this `make -C` example.